<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>165</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>863</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

QWidget
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(85, 85, 85, 241), stop:1 rgba(255, 255, 255, 255));
	color: #ffffff;
	font-weight: bold;
	border-color: #051a39;
	selection-background-color: #3b7e48;

}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QWidget" name="widget_7" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget_9" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: transparent;</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>288</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="lyrics_lab">
           <property name="font">
            <font>
             <pointsize>21</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel
{
	background-color: transparent;
	color: rgb(0, 0, 0);

}</string>
           </property>
           <property name="text">
            <string>沒有歌詞</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>287</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="maximumSize">
         <size>
          <width>803</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="song_name_lab">
        <property name="minimumSize">
         <size>
          <width>817</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>800</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel
{
	background-color: transparent;
	color: rgb(0, 0, 0);

}</string>
        </property>
        <property name="text">
         <string>歌名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <property name="minimumSize">
         <size>
          <width>800</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>800</width>
          <height>150</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: transparent;</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="add_file_btn">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QCalendarWidget QToolButton
{
  	background-color: transparent;
  	color: white;

}


QCalendarWidget QToolButton::hover
{
	background-color: #006666;
	border: 1px solid #006666;
	color: #fff;

}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="img.qrc">
             <normaloff>:/images/add-folder.png</normaloff>:/images/add-folder.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
           <property name="arrowType">
            <enum>Qt::NoArrow</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="delete_file_btn">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QCalendarWidget QToolButton
{
  	background-color: transparent;
  	color: white;

}


QCalendarWidget QToolButton::hover
{
	background-color: #006666;
	border: 1px solid #006666;
	color: #fff;

}</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="img.qrc">
             <normaloff>:/images/delete-folder.png</normaloff>:/images/delete-folder.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="popupMode">
            <enum>QToolButton::DelayedPopup</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>74</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="song_file_combobox">
           <property name="minimumSize">
            <size>
             <width>128</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QComboBox
{
    background-color:rgb(255, 255, 255);
    border: 1px solid;
    padding-left: 6px;
    color: #000;
    height: 20px;

}


QComboBox::disabled
{
	background-color: #404040;
	color: #656565;
	border-color: #051a39;

}


QComboBox:hover
{
    background-color: #f6f6f6;

}


QComboBox:on
{
    background-color: #f8f6f0;

}


QComboBox QAbstractItemView
{
    background-color: #383838;
    color: #ffffff;
    border: 1px solid black;
    selection-background-color: #488f44;
    selection-color: #ffffff;
    outline: 0;

}


QComboBox::drop-down
{
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;

}


QComboBox::down-arrow
{
	background-color: #383838;	
    image: url(://arrow-down.png) 16px 16px;
    width: 18px;
    height: 18px;
	border-left: 1px solid black;

}</string>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>新增項目</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>33</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="previous_btn">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton:hover{background-color:transparent;}
QCalendarWidget QToolButton
{
  	background-color: transparent;
  	color: white;

}


QCalendarWidget QToolButton::hover
{
	background-color: #006666;
	border: 1px solid #006666;
	color: #fff;

}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="img.qrc">
             <normaloff>:/images/previous.png</normaloff>:/images/previous.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="play_pause_btn">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton:hover{background-color:transparent;}
QCalendarWidget QToolButton
{
  	background-color: transparent;
  	color: white;

}


QCalendarWidget QToolButton::hover
{
	background-color: #006666;
	border: 1px solid #006666;
	color: #fff;

}</string>
           </property>
           <property name="text">
            <string>pause</string>
           </property>
           <property name="icon">
            <iconset resource="img.qrc">
             <normaloff>:/images/pause.png</normaloff>:/images/pause.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="next_btn">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton:hover{background-color:transparent;}
QCalendarWidget QToolButton
{
  	background-color: transparent;
  	color: white;

}


QCalendarWidget QToolButton::hover
{
	background-color: #006666;
	border: 1px solid #006666;
	color: #fff;

}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="img.qrc">
             <normaloff>:/images/next.png</normaloff>:/images/next.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>49</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="current_time_lab">
           <property name="styleSheet">
            <string notr="true">QLabel
{
	background-color: transparent;
	color: rgb(0, 0, 0);

}</string>
           </property>
           <property name="text">
            <string>00:00:00</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">QLabel
{
	background-color: transparent;
	color: rgb(0, 0, 0);

}</string>
           </property>
           <property name="text">
            <string>/</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="total_time_lab">
           <property name="styleSheet">
            <string notr="true">QLabel
{
	background-color: transparent;
	color: rgb(0, 0, 0);

}</string>
           </property>
           <property name="text">
            <string>00:00:00</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="voice_btn">
           <property name="minimumSize">
            <size>
             <width>57</width>
             <height>36</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QCalendarWidget QToolButton
{
  	background-color: transparent;
  	color: white;

}


QCalendarWidget QToolButton::hover
{
	background-color: #006666;
	border: 1px solid #006666;
	color: #fff;

}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="img.qrc">
             <normaloff>:/images/small_sound.png</normaloff>:/images/small_sound.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="voice_slider">
           <property name="minimumSize">
            <size>
             <width>39</width>
             <height>9</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>79</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"> QSlider::handle:horizontal{ 

		width:  5px; 
		height: 50px; 
		margin-top: -20px; 
		margin-left: 0px; 
		margin-bottom: -20px; 
		margin-right: 0px; 
 		
	background-color: rgb(80, 80, 80);
} 
</string>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
